kolla/node_custom_config/neutron/ml2_conf.ini

```diff
@@ -2,11 +2,10 @@
 mechanism_drivers = {% if enable_neutron_wireguard | bool %}wireguard,{% endif %}openvswitch,genericswitch,{% if enable_ironic_neutron_agent | bool %}baremetal,{% endif %}l2population
 
 [ml2_type_flat]
-{% if enable_ironic | bool %}
+# Allow creating flat networks against any physical provider network
+# (in practice this isn't always valid, but Kolla Ansible by default is opinionated
+# about what the provider networks should be named.)
 flat_networks = *
```

Comment:
This is just a "loosening" on the restrictions for what physnets can hold a network without a segmentation ID. Only admins can create such networks, so this is probably fine to leaver as `*`